song/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
This is one of the default files that django creates itself. If you're not using any of them, you should delete it and those should not be pushed.